src/test/resources/db/scripts/truncate.sql

```diff
@@ -18,7 +18,6 @@ TRUNCATE table errand_number_sequence;
 TRUNCATE table label;
 TRUNCATE table communication;
 TRUNCATE table communication_attachment;
-TRUNCATE table communication_attachment_data;
```

Comment:
Should we drop this table in a flyway-script as well?

(or create a subtaks for this if you want to drop it after deploya and verification?)

Reply:
Was considering including dropping it in the flayway script, but felt like dropping it after verification was a safer way to go about things. Will add subtask